app/services/data/get-team-caseload.js

```diff
@@ -0,0 +1,27 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+module.exports = function (id) {
+  var table = 'team_caseload_overview'
+  var whereObject = {}
+  if (id !== undefined) {
```

Comment:
Should this be the case for team? This was included in the other queries to account for national level not having an id, but I don't think we ever want to query the team table with no filter as we will get incorrect results.